### DISCUSSION TITLE
TuYa HY08WE

### DISCUSSION THREAD

**Discussion by v1dlak** [Koenkk/zigbee2mqtt#5405] [Devices]
[TuYa HY08WE](https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/devices/HY08WE.md) from documentation is [this](https://www.banggood.com/Floureon-HY08WE-2-Electric-Heating-Thermostat-LCD-Display-Digital-Thermometer-p-1398465.html) "With Wifi Function"?

Is this really possible connect via zigbee? I didn't find manual for pairing :disappointed:

**Comment by 1subu** [comment]
If you have a device with zigbee, press Button OK and SET together for 5 seconds
